woocommerce-services.php

```diff
@@ -473,9 +479,9 @@ public function init_core_wizard_shipping_config() {
 					if (
 						1 === count( $zone['zone_locations'] ) &&
 						'country' === $zone['zone_locations'][0]->type &&
-						$store_country === $zone['zone_locations'][0]->code
+						WC()->countries->get_base_country() === $zone['zone_locations'][0]->code
```

Comment:
Let's get the base country outside of the loop so we're not retrieving it more than necessary.